src/catalyst-toolbox/snapshot-lib/src/sve.rs

```diff
@@ -48,9 +49,23 @@ impl Snapshot {
                 acc
             });
 
+        // Because of multiple registrations to the same voting key,  we can only
+        // filter once all registrations for the same key are known.
+        // `min_stake_threshold` is the minimum stake for all registrations COMBINED.
+        inner.retain(|_, regs| {
```

Comment:
Seems it is missing `total_rejected_registrations` correct incrementing in this place

Reply:
We don't do anything with that information anyway.